projects/demo/src/pages/recipes/placeholder/examples/2-phone/mask.ts

```diff
@@ -18,6 +18,7 @@ export const {
      * });
      */
     removePlaceholder,
+    plugins,
     ...placeholderOptions
 } = maskitoWithPlaceholder(PLACEHOLDER);
```

Comment:
```suggestion
} = maskitoWithPlaceholder(PLACEHOLDER, true);
```
?

Reply:
No, because in this case we have custom logic for focus/blur to remove `+1` if it's the only remaining value and to set `+1 (` upon focusing empty input.